pkg/reconciler/openshift/tektonresult/extension.go

```diff
@@ -48,9 +63,37 @@ func (oe openshiftExtension) Transformers(comp v1alpha1.TektonComponent) []mf.Tr
 func (oe openshiftExtension) PreReconcile(ctx context.Context, tc v1alpha1.TektonComponent) error {
 	return nil
 }
-func (oe openshiftExtension) PostReconcile(context.Context, v1alpha1.TektonComponent) error {
-	return nil
+func (oe openshiftExtension) PostReconcile(ctx context.Context, tc v1alpha1.TektonComponent) error {
+
+	manifest, err := postManifest()
+	if err != nil {
+		return err
+	}
+	err = oe.installerSetClient.PostSet(ctx, tc, manifest, filterAndTransform())
+	return err
 }
-func (oe openshiftExtension) Finalize(context.Context, v1alpha1.TektonComponent) error {
-	return nil
+
+func postManifest() (*mf.Manifest, error) {
+	koDataDir := os.Getenv(common.KoEnvKey)
+	manifest := &mf.Manifest{}
+
+	resultsRbac := filepath.Join(koDataDir, "openshift", "tekton-results-route-rbac")
+	if err := common.AppendManifest(manifest, resultsRbac); err != nil {
```

Comment:
@khrm this is a static manifests and never changed. do not fetch from disk on each reconcile loop. that may degrade the overall performance.

Reply:
Changed to static manifests.